dockerfiles/web/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG TRAVIS_PYTHON_VERSION
 ARG DOCKER_BUILD_ENV
-FROM python:${TRAVIS_PYTHON_VERSION:-3.8}-buster
+FROM python:${TRAVIS_PYTHON_VERSION:-3.11}-buster
```

Comment:
Ideally this change is merged with this PR, but I thought it'd be good to test these changes with 3.11 in the mean time: https://github.com/lucyparsons/OpenOversight/pull/962